pkg/ingestor/parser/common/scanner/scanner.go

```diff
@@ -115,7 +118,64 @@ func purlsLicenseScanWithClient(
 }
 
 func PurlsDepsDevScan(ctx context.Context, purls []string) ([]assembler.CertifyScorecardIngest, []assembler.HasSourceAtIngest, error) {
```

Comment:
looks like this is set to false as its not part of the flags yet? By default it should be false but there is no flag to enable it.

Reply:
I think this should be implemented in : https://github.com/guacsec/guac/pull/2265/files#diff-fc14c5bdd69817e3f9b1f331130f5055f448774eb64db13c0ef23a8e882cd1d4R74-R76? Does that address your comment?

Reply:
did we need to add it here: https://github.com/guacsec/guac/blob/main/pkg/cli/store.go